doc/man3/flux_reactor_create.adoc

```diff
@@ -80,11 +80,9 @@ RETURN VALUE
 `flux_reactor_create()` returns a flux_reactor_t object on success.
 On error, NULL is returned, and errno is set appropriately.
 
-`flux_reactor_run()` returns zero on normal termination.
-On failure, termination by `flux_reactor_stop_error()` -1 is returned
-with errno set.  Additionally if either FLUX_REACTOR_NOWAIT or
-FLUX_REACTOR_ONCE is set and there are still watchers registered -1 is
-returned and errno set to EWOULDBLOCK.
+`flux_reactor_run()` returns the number of active watchers on
+successful termination.  On failure termination by
```

Comment:
A comma was dropped here, ("On failure, termination by" vs "On failure termination by"), was that done on purpose?

I'm not sure how to improve the wording (or if it needs it), maybe

 > If the reactor was terminated by `flux_reactor_stop_error()`, -1 is returned with errno set